packages/common/src/types.ts

```diff
@@ -79,10 +79,17 @@ export interface AppMetadata {
   /* App name */
   name: string
 
-  /* SVG icon string or image url, with height set to 100% */
-  icon: string
-
-  /* SVG logo (icon and text) string or image url, with width set to 100% */
+  /* An SVG icon string or image url, with height set to 100% 
```

Comment:
May want to add these notes to the core readme and the core docs pages

Reply:
added to readme! 🙌🏾  @Adamj1232 core docs update would be a separate PR ya? since it isn't included in this branch